src/exchanges.js

```diff
@@ -106,6 +106,7 @@ var Publisher = function(entries, exchangePrefix, connectionFunc, options) {
 
             // Handle errors
             if (err) {
+              err.entryName = entry.name;
```

Comment:
`entryName` is bad... this would be called `methodName` or something...

But even better let's add:
```js
err.exchange = exchange;
err.routingKey = routingKey;
err.message = payload;
err.ccRoutingKeys = CCs;
```

Reply:
I think we want to call it err.payload since `err.message` is a thing in js already?

Reply:
fair point :) hehe `message` would be a bad name :)